tigerpath/majors_and_certificates/majors/ARC_2018.json

```
@@ -0,0 +1,194 @@
+      "completed_by_semester": 4,
+      "course_list": [
+        "ARC 203",
+        "ARC 204"
+      ]
+    },
+    {
+      "name": "Departmentals",
+      "max_counted": 1,
+      "min_needed": "ALL",
+      "explanation": "In addition to the general prerequisites and the requirements for independent work, each student is required to complete ten courses in three related areas: history and theory, technology, and design. The History and Theory distribution requires six courses: three History and Theory of Architecture courses, two of which are ARC 308 and ARC 403; two courses in History and Theory of Urbanism and Landscape; and at least two upper level courses to be taken in the Department of Art and Archaeology. The Technology distribution requires two courses, one of which is ARC 311: Building Science and Technology–Building Systems. The Design Seminar distribution requires ARC 206.\n\nArchitecture concentrators take the following sequence of courses in their junior and senior year: Junior Studio, which consists of ARC 350 and ARC 351, and two required history and theory courses: ARC 308: History of Architectural Theory (taken in the fall of the junior year) and ARC 403: Topics in the History and Theory of Architecture in the fall semester of the senior year. These courses introduce methodologies of historical analysis and research, the literature of the field, and varieties of architectural writing.\n\nIn the fall semester of their senior year, students are required to enroll in ARC 404: Advanced Design Studio. ARC 404 is centered around independent design projects in that synthesize students’ training and interests and investigate new approaches to representation.",
+      "req_list": [
+        {
+          "name": "History and Theory of Architecture",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": "Three History and Theory of Architecture courses, two of which are ARC 308 and ARC 403",
+          "req_list": [
+            {
+              "name": "Required Courses",
+              "max_counted": 1,
+              "min_needed": "ALL",
+              "explanation": null,
+              "course_list": [
+                "ARC 308",
+                "ARC 403"
+              ]
+            },
+            {
+              "name": "Electives",
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "<i>Editor's Note: This does not represent a comprehensive list of courses that fulfill this requirement as courses change from semester to semester.</i>",
+              "course_list": [
+                "AMS 322/URB 322/ARC 326/AAS 320",
+                "ARC 302/ART 347",
+                "ARC 308/ART 328",
+                "ARC 322",
+                "ARC 489",
+                "ARC 492/URB 492/ENV 492",
+                "ART 102/ARC 209",

```

It seems like these are two separate courses.
```suggestion
                "ART 102/ARC 102",
                "ART 201/ARC 209",
```